### ISSUE TITLE
Theme Slaughter: Add editing

### ISSUE THREAD

**Issue by mikekasprzak** [ludumdare/ludumdare#325]
Like it used to be, it would be nice if you could click on a theme in your Previous Themes list, and change your vote.

**Comment by noah-potter** [comment]
Would this work like a toggle? i.e. clicking on the 'x' symbol would immediately change the vote to YES and show the checkmark?

**Comment by povrazor** [comment]
No, it would change what you're currently voting on (as it used to do).

**Comment by noah-potter** [comment]
Gotcha. I haven't used the old site very much so I'll be asking basic questions about how things worked, sorry. I'd like to help but I can't find much setup documentation. I'll keep an eye on the repo for that though.

**Comment by povrazor** [comment]
Yeah, setup is extra broken at the moment. It should be in a better state in the new year.

**Comment by gonutz** [comment]
I want to support this issue. Being unable to change my mind after I said no to a theme has bugged me the last two jams. I have even stopped slaughtering themes this time because the way I do it - probably the same as everybody else - is that I look at a theme, then think of the first idea that comes to my mind and make my decision based on that. But then two themes later I still think about the previous theme and a cool idea pops up in my head. But by then I have already rejected the theme.
For my use-case it would be enough to be able to edit the last two or three themes. Right now I can see the last 10 themes that I slaughtered so being able to just click on the checkmark/cross to toggle the decision would be great.
